examples/docker/flask/README.md

```diff
@@ -0,0 +1,31 @@
+#  App
+
+This Flask app serves a machine learning model trained on the `Walmart Recruiting - Store Sales Forecasting` dataset for predicting store sales.
+
+## Getting Started
+
+To get started with this Flask app, follow these steps:
+
+1. Run the `walmart_sales.ipynb` notebook to generate the trained model `walmart_sales_rf.joblib`.
+
+2. Create a zip file from `Dockerfile`, `app.py`, `walmart_sales_rf.joblib`, `static/` and `templates/`.
+
+3. Login to your [Ploomber Cloud](https://ploomber.io/) account.
+
+2. Follow the [steps](https://docs.cloud.ploomber.io/en/latest/apps/flask.html) for deploying a Flask application and upload the `app.zip`.
+
+## How to use
+
+Enter valid inputs for predicting the sales. Mandatory inputs are `Store`, `Dept`, `Year`, `Month` and `Week`. `Size` should be an integer and `Markdown` values are of float type.
+
+**Sample Input**:
+
+'Store':9\
```

Comment:
the app asks me for Markdown1 ... 5, what are these values?

![image](https://github.com/ploomber/doc/assets/989250/eb597007-78cb-4d0e-b0b5-ee9102b5372a)

Reply:
it should be intuitive to understand what the features mean

Reply:
I have removed these fields and included the ones in the `test.csv` only to make it simpler: https://github.com/ploomber/doc/pull/18/commits/197b4395e2e9a455a3b6ad9985fbfc35e163386f

Markdowns as per the challenge: `MarkDown1-5 - anonymized data related to promotional markdowns that Walmart is running. MarkDown data is only available after Nov 2011, and is not available for all stores all the time. Any missing value is marked with an NA.`

Let me know if I should add these fields back. I can add a brief description of markdowns.

@edublancas

Reply:
let's include descriptions for all features in the UI, along with some sample values. the idea is that users should be able to geta prediction without guessing that to enter on each field

Reply:
[fixed](https://github.com/ploomber/doc/pull/18/files#diff-c2468cf26943bd21d5d66b32ed79434e8c953cf1e779837f8648452fd75e0f16R48-R67)

Reply:
Added desc for dept no : https://github.com/ploomber/doc/pull/18/commits/b45b7c5c6dc3de42213d33d76e2a022c4a4ace41